libs/designer/src/lib/ui/templates/cards/templateCard.less

```diff
@@ -4,90 +4,96 @@
     border: 1px solid rgba(204, 204, 204, 0.50);
     border-radius: 2px;
 
-    padding: 16px 16px;
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-
+    height: 220px;
     box-shadow: 0px 1.6px 3.6px 0px rgba(0, 0, 0, 0.13), 0px 0.3px 0.9px 0px rgba(0, 0, 0, 0.11);
 }
 
-.msla-template-card-data, .msla-templates-break, .msla-template-card-connectors {
-    width: 95%;
-    display: flex;
+.msla-template-card-authored-wrapper {
+    justify-content: flex-start;
+    align-items: center;
+    width: 100%;
+    border-bottom: 1px solid #F0F0F0;
+    height: 40px;
+
+    .msla-template-card-authored {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        padding: 12px;
+    }
+    
+    .msla-template-card-authored-label {
+        align-items: flex-start;
+        margin-left: 4px;
+        color: #616161;
+    }
 }
 
-.msla-template-card-data {
+.msla-template-card-body {
+    padding: 0px 12px;
+    height: 180px;
     display: flex;
     flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    justify-content: space-around;
+}
+
+.msla-template-card-title-wrapper {
+    padding-top: 12px;
     flex: 1;
+
+    .msla-template-card-title {
+        display: -webkit-box;
+        -webkit-line-clamp: 3;
+        -webkit-box-orient: vertical;
+        overflow: hidden;
+        text-overflow: ellipsis;
+    }
 }
 
-.msla-template-card-title {
-    padding-bottom: 10px;
+.msla-template-card-footer {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;    
 }
 
 .msla-template-card-tags {
     display: flex;
     flex-direction: row;
-    color: grey;
+    padding: 12px 0px;
 
     .msla-template-card-tag {
-        margin-right: 14px;
-        color: grey;
+        margin-right: 12px;
+        color: #0F6CBD;
+        background-color: #EBF3FC;
```

Comment:
Might need to adjust this for dark mode

Reply:
Thanks for the feedback! Seems like the designs provided are not really supporting the dark mode, I will reach out to the designer to bring up this topic. The V2 changes are from V1 which have the same issue.